transactor/src/main/scala/io/mediachain/datastore/DynamoDatastore.scala

```diff
@@ -15,10 +15,12 @@ class DynamoDatastore(config: DynamoDatastore.Config)
   
   val creds = config.awscreds
   val table = config.baseTable
-  val chunkSize = 1024 * 384 // 384 KB; DynamoDB has 400KB limit
+  val chunkSize = 1024 * 384 // 384 KB; DynamoDB has 400KB limit incl field
```

Comment:
i just wanted to be explicit that the actual names of the fields count towards the limit too i.e.

`meta`, `multihash` etc